projects/demo-playwright/tests/core/dropdown/dropdown.pw.spec.ts

```diff
@@ -55,6 +55,16 @@ test.describe('Dropdown', () => {
         await expect(page).toHaveScreenshot('05-dropdown.png');
     });
 
+    test('Hosted dropdown initial width', async ({page}) => {
+        await tuiGoto(page, DemoRoute.Viewport);
+        const example = new TuiDocumentationPagePO(page).getExample('#portal');
+
+        await example.scrollIntoViewIfNeeded();
+        await example.locator('.t2').click({force: true});
```

Comment:
why `force`?

Reply:
`force` disables auto-wait checks https://playwright.dev/docs/actionability#forcing-actions
(we need to disable automatic waiting for the end of animation)